### DISCUSSION TITLE
What is mol_bs?

### DISCUSSION THREAD

**Discussion by zhs047** [Tomoki-YAMASHITA/CrySPY#42] [Q&A] [answered]
What is the mol_bs mode in struc_mode?

**Comment by Tomoki-YAMASHITA** [comment, marked as answer]
CrySPY uses pyxtal in normal molecular crystal structure generation mode (struc_mode = mol). The molecules are arranged to fit a point group at a selected Wykoff position in the space group to keep the symmetry.  (Sometimes it takes a long time to generate.)

In mol_bs mode (bs means break symmetry), dummy atoms are placed in Wykoff positions as in ordinary crystals, and then the dummy atoms are replaced by molecules without considering symmetry and rotated randomly. The structure generation is relatively fast.

**Reply by Tomoki-YAMASHITA** [reply]
I just started to make the document page.
https://tomoki-yamashita.github.io/CrySPY_doc/structure_generation/

but still under construction.
